### ISSUE TITLE
Format bug on C#

### ISSUE THREAD

**Issue by mihaipopescu** [uncrustify/uncrustify#1264]
expected:
```
Thing
    .Select(x => x > 2)
    .ToList();

var x = Thing
    .Select(x => x > 2)
    .ToList();
```

actual:

```
Thing
.Select(x => x > 2)
.ToList();

var x = Thing
    .Select(x => x > 2)
    .ToList();
```

Expected versus output diff:
---
```diff
diff --git "a/C:\\work\\external\\uncrustify\\tests\\output\\staging\\60018-UNI-18777.cs" "b/C:\\work\\external\\uncrustify\\tests\\results\\staging\\60018-UNI-18777.cs"
index 198d5be..d4e8fd8 100644
--- "a/C:\\work\\external\\uncrustify\\tests\\output\\staging\\60018-UNI-18777.cs"
+++ "b/C:\\work\\external\\uncrustify\\tests\\results\\staging\\60018-UNI-18777.cs"
@@ -1,9 +1,9 @@
 // I want to keeep the function call indented
 Thingy
-    .Select(x => x > 2)
-    .ToList();
+.Select(x => x > 2)
+.ToList();
 
 // it works with a var
 var x = Thingy
     .Select(x => x > 2)
     .ToList();
```

Debug support files:
---
[60018-UNI-18777.cs](https://gist.githubusercontent.com/mihaipopescu/d7ae31e8d91367ab2b51863e1f4b2601/raw/198d5be3d1b08590e8605cb41c50e027c6014628/60018-UNI-18777.cs)
[60018-UNI-18777.cs.log](https://gist.githubusercontent.com/mihaipopescu/d7ae31e8d91367ab2b51863e1f4b2601/raw/201c98b6a2d4871641ca2a9171b20cc0b8199b5b/60018-UNI-18777.cs.log)
[60018-UNI-18777.cs.unc](https://gist.githubusercontent.com/mihaipopescu/d7ae31e8d91367ab2b51863e1f4b2601/raw/d82c48f04e280b055def70ad723c62336b54384f/60018-UNI-18777.cs.unc)
[Uncrustify.CSharp.cfg](https://gist.githubusercontent.com/mihaipopescu/d7ae31e8d91367ab2b51863e1f4b2601/raw/1d064c1fabb76ce328dd015ed40023207de20ca1/Uncrustify.CSharp.cfg)
[UNI-18777.cs](https://gist.githubusercontent.com/mihaipopescu/d7ae31e8d91367ab2b51863e1f4b2601/raw/198d5be3d1b08590e8605cb41c50e027c6014628/UNI-18777.cs)

Disclaimer:
---
Note: This is a generated issue from our internal issue tracker.
Expect a dry description which could be insufficient to properly understand the issue.
On the other hand, the support files should be sufficient to easily reproduce and debug the issue,
but please comment if it's still unclear and I will get back with clarifications.
See main discussion thread here #1223.

**Comment by gmaurel** [comment]
fixed at dcf3c1394de6709785267c0b372c09b3345b0ef5, on Feb 10 17:46:05 2018

**Comment by CDanU** [comment]
It would be nice to always set a reference from the PR to the issue. If you have the commit sha github will show you the PR if there is one.

Click on the sha link and on the following page you will see:
```
<commitTitle:>    Adopt UT test 60018
<branch [PR-ID]>  master (#1602) 
```
